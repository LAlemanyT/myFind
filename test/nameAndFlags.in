./myfind.py ~ --name="file.txt" "wc {} -c" 
