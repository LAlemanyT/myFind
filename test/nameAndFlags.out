10 /home/a/file.txt
10 /home/a/b/file.txt
wc: /home/a/b/c/file.txt: Is a directory
0 /home/a/b/c/file.txt
